src/components/SelectCountry/index.tsx

```diff
@@ -0,0 +1,170 @@
+  const countries = () => {
+    return ["KR, Korea", "US, United States", "CN, China", "JP, Japan"];
+  };
+
+  const toggleDropDown = () => {
+    setShowDropDown(!showDropDown);
+  };
+
+  const outFoucsHandler = (e: React.FocusEvent<HTMLButtonElement>): void => {
+    if (e.currentTarget === e.target) {
+      setShowDropDown(false);
+    }
+  };
+
+  const countrySelection = (country: string): void => {
+    setSelectCountry(country);
+  };
+
+  return (
+    <>
+      <div>
+        <div>
+          {selectCountry
+            ? `You selected ${selectCountry}`
+            : "Select Your Country"}
+        </div>
+      </div>
+      <StyledButtonDiv>
+        <StyledButton
+          className={showDropDown ? "active" : undefined}
+          onClick={(): void => toggleDropDown()}
+          onBlur={(e: React.FocusEvent<HTMLButtonElement>): void =>
+            outFoucsHandler(e)
+          }
+        >
+          <div>
+            {/* {selectCountry ? "Selected: " + selectCountry : "Select ..."} */}
+            {selectCountry ? "" + selectCountry : "Select ..."}
+          </div>
+          {showDropDown && (
+            <CountryDropDown
+              countries={countries()}
+              showDropDown={false}
+              toggleDropDown={(): void => toggleDropDown()}
+              countrySelection={countrySelection}
+            />
+          )}
+        </StyledButton>
+      </StyledButtonDiv>
+    </>
+  );
+};
+
+// function SelectCountry() {
+//   // const [isOpenCountry, setIsOpenCountry] = useState<boolean>(false);
+//   // const [selectedOption, setSelectedOption] = useState<string>("");
+
+//   // const dropClickCountry = () => {
+//   //   setIsOpenCountry(!isOpenCountry);
+//   // };
+
+//   // const selectOnChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+//   //   const value = e.target.value;
+//   //   setSelectedOption(value);
+//   // };
+// {
+//   <Select
+//     placeholder={"countries"}
+//     options={options}
+//     onChange={selectOnChange}
+//   />;
+// }
+
+export default SelectCountry;
+
+const StyledSelectBox = styled.div`
+  position: relative;
+  width: 200px;
+  padding: 8px;
+  border-radius: 12px;
+  background-color: #fff;
+  align-self: center;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  cursor: pointer;
+  &::before {
+    content: "∨";
+    position: absoulte;
+    top: 1px;
+    right: 8px;
+    color: #49c181;
+    font-size: 1.25rem;
+  }
+`;
+
+const StyledButton = styled.button`
+  position: relative;
+  padding: 0.375rem 0.88rem;
+  color: ${({ theme }) => theme.color.white};
+  background-color: ${({ theme }) => theme.color.grey};
+  font-weight: ${({ theme }) => theme.fontWeight.normal};
+  text-align: left;
+  white-space: nowrap;
+  vertical-align: middle;
+  -webkit-user-select: none;
+  user-select: none;
+  border: 0.125rem solid transparent;
+  font-size: 0.85rem;
+  line-height: 1.5;
+  border-radius: ${({ theme }) => theme.borderRadius.imgCard};
```

Comment:
css는 비슷한 속성 끼리 묶는 게 나중에도 더 보기 좋을 것 같지 않나용 ?

```suggestion
  user-select: none;
  
  font-weight: ${({ theme }) => theme.fontWeight.normal};
  font-size: 0.85rem;
  line-height: 1.5;
  
  border: 0.125rem solid transparent;
  border-radius: ${({ theme }) => theme.borderRadius.imgCard};
```

Reply:
고치겠습니다!